tests/GridFS/BucketFunctionalTest.php

```diff
@@ -708,6 +711,21 @@ public function testExistingIndexIsReused()
         $this->assertIndexNotExists($this->chunksCollection->getCollectionName(), 'files_id_1_n_1');
     }
 
+    public function testDanglingOpenWritableStream()
+    {
+        $path = __DIR__ . '/../../vendor/autoload.php';
+        @exec(
+            <<<CMD
+php -r "require '$path'; class_exists(MongoDB\Operation\FindOne::class); \\\$stream = (new MongoDB\Client)->test->selectGridFSBucket()->openUploadStream('filename', ['disableMD5' => true]);" 2>&1
+CMD,
```

Comment:
Why is `class_exists(MongoDB\Operation\FindOne::class);` necessary?

IIRC, one of the relevant bits of [PHPLIB-345](https://jira.mongodb.org/browse/PHPLIB-345) was that if the class wasn't _already_ loaded, then we'd see an error when our GridFS code attempted to load it during shutdown. I would think that triggering autoloading ahead of time might defeat the point of a regression test.

Reply:
This is left over from a previous attempt at fixing this. I thought I had removed it, but apparently not.

Reply:
Removed.